<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>2000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ChessBoard">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>700</width>
      <height>700</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/720px-Chessboard480.svg.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="KnightW">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/horseya.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="PawnB2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/pawnBlacks.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="BishopW">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/Bishopaya.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="PawnB1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/pawnBlacks.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Bishop">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>340</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bishop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>270</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Starting Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>310</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Target Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_result">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>510</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_Src">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>273</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_Dest">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>310</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>420</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pawn Position</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_Pwn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>420</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addPwn">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>410</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="PawnB3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/pawnBlacks.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="PawnB4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/pawnBlacks.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="PawnB5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/pawnBlacks.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="PawnB6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/pawnBlacks.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="PawnB7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/pawnBlacks.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="PawnB8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/pawnBlacks.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="no1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>63</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/1g.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="no2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>63</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/2g.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="no3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>63</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/3g.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="no4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>63</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/4g.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="no5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>63</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/5g.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="no6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>63</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/6g.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="no7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>63</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/7g.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="no8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>63</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/resources/img/8g.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Calculate">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>140</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_knight">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>340</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Knight</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2000</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
